library/menu/src/PatientMenuRole.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * MainMenuRole class.
```

Comment:
change comment to PatientMenuRole class

Reply:
Of course , just the same old price of copy&paste :)